script/scaffold/__main__.py

```diff
@@ -98,14 +130,13 @@ def main() -> int:
             "--integration",
             info.domain,
         ],
-        check=True,
+        info,
     )
-    print()
 
     if args.develop:
         print("Running tests")
-        print(f"$ python3 -b -m pytest -vvv tests/components/{info.domain}")
```

Comment:
The full pytest command is no longer printed now. Is that intended?

Reply:
I removed to make it consistent with all the others, if we think it's useful to have this output we could add `run_process` so it's consistent. Thoughts?

Reply:
Yeah, I think it's nice to see exactly what commands are being run.

Reply:
Done, tweaked the output while there too.